### DISCUSSION TITLE
changing default directory of zim

### DISCUSSION THREAD

**Discussion by seri3s** [zim-desktop-wiki/zim-desktop-wiki#2571] [Q&A]
Hi, 

I'm using zim 0.75.3 on PI-OS 64-bit.

The default directory is fixed to ~NoteBooks by .config/zim . When I have a different NoteBooks in a different location, how do I open that NoteBooks?

Can I open it by command-line such like as below:
$ zim --root-url Different-NoteBooks

I guess --root-url is not  for this purpose, but I hope this kind of option to open a different NoteBooks, without replacing all directory to the new one.

I do not know newer zim version of 1.7.19 which is far beyond my version of 0.75.3
Please advise me this.

Thanks and Regards,
Shigeru

**Comment by jaap-karssenberg** [comment]
You can simply open a folder as a Zim notebook using 'zim /path/to/folder'.
Also you can open any folder from the open notebook dialog.

Does that answer your question?



Op vr 9 feb. 2024 23:53 schreef shigerusz ***@***.***>:
